numpy/core/fromnumeric.py

```diff
@@ -1385,7 +1385,7 @@ def squeeze(a, axis=None):
     try:
         squeeze = a.squeeze
     except AttributeError:
-        return _wrapit(a, 'squeeze')
+        squeeze = functools.partial(_wrapit, obj=a, method='squeeze')
```

Comment:
There's no need to use partial here - you could just call `_wrapit(a, 'squeeze', axis=axis)`, since at this point the codepath goes through `ndarray.squeeze`, which natively supports `axis`

Reply:
there I go overcomplicating things again... :)